src/content-helper/dashboard-page/pages/dashboard/page-component.tsx

```
@@ -1,13 +1,18 @@
+import { PageContainer as PageContainer, PageBody } from '../../components';
```

Biome

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)